convo.go

```diff
@@ -51,18 +87,32 @@ func (c *Conversation[X]) CmdAskProjectName() loop.Cmd {
 }
 
 func (c *Conversation[X]) HandleSubstreamsConsumptionChoice(value string) loop.Cmd {
+	// Try to get actual network and module information
+	network, outputModule, hasInfo := c.GetNetworkAndModule()
+	
+	// Prepare the values to use in commands
+	var endpoint, module string
+	if hasInfo {
+		endpoint = NetworkToEndpoint(network)
+		module = outputModule
+	} else {
+		// Fallback to placeholders if we can't determine the values
+		endpoint = "{endpoint}"
+		module = "{output_module}"
```

Comment:
Let's use `<>` for placeholder when they are arguments to a command
```suggestion
		endpoint = "<endpoint>"
		module = "<output_module>"
```